src/Microsoft.OData.Edm/Csdl/Semantics/IEdmReferentialConstraint.cs

```diff
@@ -1,26 +0,0 @@
-﻿//---------------------------------------------------------------------

```

Comment:
Will add consolidated descriptions of the commits into the overall description when we do a squash PR.